bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/protocol/leap/LeapMessageParserCallbacks.java

```diff
@@ -15,11 +15,7 @@
 import java.util.List;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.Area;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.ButtonGroup;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.Device;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.OccupancyGroup;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.ZoneStatus;
+import org.openhab.binding.lutron.internal.protocol.leap.dto.*;
```

Comment:
Please revert as it introduces [checkstyle warnings](https://ci.openhab.org/job/PR-openHAB-Addons/17008/Static_20Code_20Analysis_20Report/)

Reply:
resolved